Formula/mame.rb

```diff
@@ -47,10 +55,39 @@ def install
       man1.install "build/man/MAME.1" => "mame.1"
     end
     pkgshare.install %w[artwork bgfx hash ini keymaps plugins samples uismall.bdf]
+
+    if build.with? "tools"
+      bin.install %w[
+        aueffectutil castool chdman floptool imgtool jedutil ldresample
+        ldverify nltool nlwav pngcmp regrep romcmp src2html srcclean unidasm
+      ]
+      bin.install "split" => "rom-split"
+      man1.install Dir["docs/man/*.1"]
+    end
   end
 
   test do
     assert shell_output("#{bin}/mame -help").start_with? "MAME v#{version}"
     system "#{bin}/mame", "-validate"
+
+    if build.with? "tools"
+      # system "#{bin}/aueffectutil" # segmentation fault
```

Comment:
Because it's part of MAME's tools.
But since Homebrew won't accept unstable formulae, and some MAME tools never get working, I'll remove the SEGFAULT binary in next few days.